src/Microsoft.Performance.SDK/Processing/ProcessingSource.cs

```diff
@@ -197,6 +198,30 @@ public virtual ProcessingSourceInfo GetAboutInfo()
             return processor;
         }
 
+        /// <inheritdoc />
+        public ICustomDataProcessor CreateProcessor(
```

Comment:
>CreateProcessor [](http://example.com/codeflow?start=36&length=15)

We haven't used the <exception></exception> doc comments as consistently as we should, but we should try.

Looks like several possible exceptions here.

Reply:
FYI, we've typically put expected exception comments on interfaces where appropriate. I have mixed feelings about this because an implementation might throw those listed exceptions, or possibly others. But interfaces on learn.microsoft.com do list possible exceptions, so I'm guessing that's why we've done it.